Adding DOAP file draft

cottage14 opened the pull request:
I have created a DOAP file, per a discussion on board@a.o that reminded me that we do not seem to have one for Royale. I did not add information about an 'implemented standard' because I do not know what standard applies to Royale.

I invite suggestions to improve this information. Once this file is in good-enough shape, I will register it per the instructions at https://projects.apache.org/create.html. I will undertake to update it as necessary (mainly when we release a new version).

cottage14 committed:
Adding DOAP file draft

I have created a DOAP file, per a discussion on board@a.o that reminded me that we do not seem to have one for Royale. I did not add information about an 'implemented standard' because I do not know what standard applies to Royale.

I invite suggestions to improve this information. Once this file is in good-enough shape, I will register it per the instructions at https://projects.apache.org/create.html. I will undertake to update it as necessary (mainly when we release a new version).

Harbs commented:
I don't know what a DOAP file is supposed to look like, but I assume the HTML at the bottom with error information is not correct?

cottage14 commented:
You are probably right! The ASF provides a tool for creating the DOAP file (
https://projects.apache.org/create.html) and I just filled in the form,
generated the file, and grabbed the result.

Since the DOAP never actually meets HTML (other processes harvest its
information), I will pull that error information from our file.

a


On Wed, Sep 6, 2023 at 3:46 AM Harbs ***@***.***> wrote:

> I don't know what a DOAP file is supposed to look like, but I assume the
> HTML at the bottom with error information is not correct?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/royale-asjs/pull/1230#issuecomment-1707765031>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC3RFXK2WLS6ACJHPTWDUILXZAL3XANCNFSM6AAAAAA4MDBEHU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

cottage14 committed:
Update doap_Royale.rdf

The tool provided by the ASF generated a DOAP file with a range of errors. After running it through the W3C Validator (https://www.w3.org/RDF/Validator/rdfval) a bunch of times, I was able to isolate and repair the various errors.

cottage14 committed:
Update doap_Royale.rdf

continuing to remove extraneous material generated by the DOAP creation too.

cottage14 commented:
I think I have scraped all the bogus stuff out of our DOAP file. It now
passes review in the W3C validation tool (https://www.w3.org/RDF/Validator/
).

Unless someone spots something else, I would like to commit the updated
file and submit it to the ASF. They may/will find other issues, but they
can't find them until we have a file for them to look at.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Wed, Sep 6, 2023 at 9:26 AM Andrew Wetmore ***@***.***> wrote:

> You are probably right! The ASF provides a tool for creating the DOAP file
> (https://projects.apache.org/create.html) and I just filled in the form,
> generated the file, and grabbed the result.
>
> Since the DOAP never actually meets HTML (other processes harvest its
> information), I will pull that error information from our file.
>
> a
>
>
> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
> Virus-free.www.avast.com
> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
> <#m_4624483819144522354_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>
> On Wed, Sep 6, 2023 at 3:46 AM Harbs ***@***.***> wrote:
>
>> I don't know what a DOAP file is supposed to look like, but I assume the
>> HTML at the bottom with error information is not correct?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/apache/royale-asjs/pull/1230#issuecomment-1707765031>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AC3RFXK2WLS6ACJHPTWDUILXZAL3XANCNFSM6AAAAAA4MDBEHU>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Andrew Wetmore
>
> Editor, Moose House Publications <https://moosehousepress.com>
> Editor-Writer, The Apache Software Foundation <https://apache.org>
>


-- 
Andrew Wetmore

Editor, Moose House Publications <https://moosehousepress.com>
Editor-Writer, The Apache Software Foundation <https://apache.org>

Harbs commented:
Looks good to my uneducated eye...

aharui commented:
I thought I put together a valid DOAP for Flex.  I don’t have time to look and compare (and I’m sure it is out-of-date), but it should be out there to use as a reference point.

-Alex

From: Harbs ***@***.***>
Reply-To: apache/royale-asjs ***@***.***>
Date: Wednesday, September 6, 2023 at 8:53 AM
To: apache/royale-asjs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [apache/royale-asjs] Adding DOAP file draft (PR #1230)


EXTERNAL: Use caution when clicking on links or opening attachments.



Looks good to my uneducated eye...

—
Reply to this email directly, view it on GitHub<https://github.com/apache/royale-asjs/pull/1230#issuecomment-1708654692>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAQIK6OA2LG3UYKQCXGLBFDXZCMALANCNFSM6AAAAAA4MDBEHU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

cottage14 commented:
Yes, the Flex DOAP file is in place and is a good model.

a

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Wed, Sep 6, 2023 at 1:58 PM aharui ***@***.***> wrote:

> I thought I put together a valid DOAP for Flex. I don’t have time to look
> and compare (and I’m sure it is out-of-date), but it should be out there to
> use as a reference point.
>
> -Alex
>
> From: Harbs ***@***.***>
> Reply-To: apache/royale-asjs ***@***.***>
> Date: Wednesday, September 6, 2023 at 8:53 AM
> To: apache/royale-asjs ***@***.***>
> Cc: Subscribed ***@***.***>
> Subject: Re: [apache/royale-asjs] Adding DOAP file draft (PR #1230)
>
>
> EXTERNAL: Use caution when clicking on links or opening attachments.
>
>
>
> Looks good to my uneducated eye...
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/apache/royale-asjs/pull/1230#issuecomment-1708654692>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AAQIK6OA2LG3UYKQCXGLBFDXZCMALANCNFSM6AAAAAA4MDBEHU>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/royale-asjs/pull/1230#issuecomment-1708769677>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC3RFXIVW57W7A2QY7ZYUHTXZCTVBANCNFSM6AAAAAA4MDBEHU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Andrew Wetmore

Editor, Moose House Publications <https://moosehousepress.com>
Editor-Writer, The Apache Software Foundation <https://apache.org>
